app/models/solidus_subscriptions/subscription.rb

```diff
@@ -241,6 +241,13 @@ def billing_address_to_use
       billing_address || user.bill_address
     end
 
+    def clear_past_installments
+      installments.unfulfilled.each do |installment|
+        installment.actionable_date = nil
+        installment.save
```

Comment:
We should probably use `save!`, to make sure nothing goes wrong when clearing the old installments.

Reply:
Changed it. I'm checking if it raises errors in some specs.